src/gtest/test_bip32.cpp

```diff
@@ -36,6 +42,22 @@ struct TestVector {
     }
 };
 
+class TestBip32 : public Test
+{
+public:
+    void SetUp() override
```

Comment:
you have to set chain params only once... 
so change this to:
static void SetUpTestSuite()
{
   ....
}
This will be executed  before all tests in this fixture once.

Reply:
Fixed. Thanks